content/en/docs/reference/_index.md

```diff
@@ -16,11 +18,17 @@ This section of the Kubernetes documentation contains references.
 
 <!-- body -->
 
-## API Reference
+## Standardized Glossary
+
+* [Glossary](/docs/reference/glossary/) -  a comprehensive, standardized list of Kubernetes terminology
+
+
```

Comment:
These API-related list items (related to "API overview, access, reference") need a heading (not "Standardized Glossary").